### PR TITLE
fix cirucular refrence in arg and indentation issue

### PR DESCRIPTION
**Pull request by pruan-rht**
indentation was wrong
```
    def ready?(user: nil, quiet: false, cached: false)
    	res = get(user: user, quiet: quiet)
    	res[:success] = res[:parsed]["status"]["phase"] == "Succeeded" && res[:parsed]["status"]["reason"] == "InstallSucceeded" if res[:success]
    	return res
    end
```

### CODE REVIEW

File: lib/openshift/cluster_service_version.rb [hunk 1]
```diff
@@ -4,15 +4,12 @@ module BushSlicer
   class ClusterServiceVersion < ProjectResource
     RESOURCE = "clusterserviceversions"
 
-    # @return [BushSlicer::ResultHash] with :success depending on 
+    # @return [BushSlicer::ResultHash] with :success depending on
     # status phase=Succeeded and reason=InstallSucceeded;
-    def ready?(user: user, quiet: false, cached: false)
-    	res = get(user: user, quiet: quiet)
-    	if res[:success]
```
Comment (akostadinov) [inline lib/openshift/cluster_service_version.rb:11]: this has a purpose. If the `get` call failed then there is no `:parsed` element. In the new version, if `get`  failed, then no method for Nil class will be thrown.

File: lib/openshift/cluster_service_version.rb [hunk 2]
```diff
@@ -4,15 +4,12 @@ module BushSlicer
...
-    		res[:success] = 
-    			res[:parsed]["status"]["phase"] == "Succeeded" && res[:parsed]["status"]["reason"] == "InstallSucceeded"
-    	end
-    	return res
+    def ready?(user: nil, quiet: false, cached: false)
+      res = get(user: user, quiet: quiet)
+      res[:success] = res[:parsed]["status"]["phase"] == "Succeeded" && res[:parsed]["status"]["reason"] == "InstallSucceeded" if res[:success]
```
Comment (akostadinov) [inline lib/openshift/cluster_service_version.rb:11]: Is there any change in logic? When `if` is at the end, it's not very visible. Can we keep using `if` on separate lines?

Comment (pruan-rht) [reply lib/openshift/cluster_service_version.rb]: no change in the logic, it was originally to fix the circular reference warning, but then I noticed the indentation issue (which only shows up in `cat`)

**Comment by akostadinov** [comment]
Thank you.

/lgtm

File: lib/openshift/cluster_service_version.rb (hunk 2 above)
Comment (pruan-rht) [reply lib/openshift/cluster_service_version.rb]: @akostadinov anyway, I've put back the original formatting with the indentation fix

**Pull request merged**